setup.py

```diff
@@ -16,7 +16,8 @@
 install_requires = [
     'six',
     # >= 8 9s because of breaking API change
-    'html5lib>=0.99999999',
+    # the 'pre' suffix is needed for supporting '1.0b*' versions
```

Comment:
Ahh... interesting. Do we know if that's dependent on pip versions or does it work with old pip versions, too?